doc/building.rst

```diff
@@ -128,9 +128,20 @@ Then, in Terminal:
 
    .. code-block:: bash
 
-      % brew install git qt qtkeychain cmake openssl glib cmocka
+      % brew install git qt5 qtkeychain cmake openssl glib cmocka
 
-5. Certain Homebrew packages are not automatically linked in places where
+.. note:: Currently the build process will fail if you have both Qt 5
+   and Qt 6 installed. In order to remove Qt 6, run the following:
+
+   .. code-block:: bash
+
+      % brew remove qt6
+
+   Unfortunately, removing Qt 6 will break any other Homebrew packages
+   that depend on it. An update fixing the build process so that it will
+   work with both packages installed is forthcoming.
+
+1. Certain Homebrew packages are not automatically linked in places where
    the build scripts can find them, so you can create a shell-profile script
    that will find and load them dynamically when you run a build:
 
```

Comment:
We should probably add QtKeychain to the commands below:

`echo 'export Qt5Keychain_DIR=$(brew --prefix qtkeychain)/0.13.2/' >> ~/.nextcloud_build_variables`

Reply:
Or, why don't we just provide a CMake command here? Seems like it would be a lot easier to just say "go run this in the build folder":

`cmake .. -DOPENSSL_ROOT_DIR=$(brew --prefix openssl) -DQt5_DIR=$(brew --prefix qt5)/lib/cmake/Qt5/ -DQt5Keychain_DIR=$(brew --prefix qtkeychain)/lib/cmake/ -DQt5LinguistTools_DIR=$(brew --prefix qt5)/lib/cmake/Qt5LinguistTools/`

Reply:
Or, why don't we just provide a CMake command here? Seems like it would be a lot easier to just say "go run this in the build folder":

`cmake .. -DOPENSSL_ROOT_DIR=$(brew --prefix openssl) -DQt5_DIR=$(brew --prefix qt5)/lib/cmake/Qt5/ -DQt5Keychain_DIR=$(brew --prefix qtkeychain)/lib/cmake/ -DQt5LinguistTools_DIR=$(brew --prefix qt5)/lib/cmake/Qt5LinguistTools/`